js/main.js

```diff
@@ -20,3 +20,25 @@ const lineLenght = function (line, maxLenght) {
 
 lineLenght('', 130 );
 
+const getRandomArrayElement = (elements) => elements[getRandomIntInclusive(0, elements.length - 1)];
+
+const ID = Array.from({length: 25}, (_, index) => index + 1);
```

Comment:
Получаем массив, нужно это обозначить
```suggestion
const IDs = Array.from({length: 25}, (_, index) => index + 1);
```